docs/dark-bckg.svg

```diff
@@ -0,0 +1,30 @@
+<svg width="344" height="126" viewBox="0 0 344 126" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Is this file being used? Also, this same file seems to be named `phylum-blue.svg` elsewhere, so why is it `dark-bckg.svg` here?